bulletin-buddy-authentication-service/src/main/java/com/github/bulletinboard/controllers/AuthenticationController.java

```diff
@@ -18,7 +18,8 @@ public boolean addUser(@RequestBody User user) {
     }
 
     @GetMapping()
-    public boolean authenticateUser(@RequestBody User user) {
+    public boolean authenticateUser(@RequestParam("name") String name, @RequestParam("passwd") String passwd) {
```

Comment:
Just as a style note, I would use "password" instead of "passwd"

Reply:
fixed